addon/components/text-area.js

```diff
@@ -1,8 +1,28 @@
-import Component      from 'ember-component';
-import textInputMixin from 'ember-railio-form-components/mixins/text-input-mixin';
+import Component        from 'ember-component';
+import textInputMixin   from 'ember-railio-form-components/mixins/text-input-mixin';
+import formFieldOptions from 'ember-railio-form-components/mixins/form-field-options'; //eslint-disable-line
 
-export default Component.extend(textInputMixin, {
+import get from 'ember-metal/get';
+
+function textAreaAjust(element) {
+  element.style.height = '1px';
```

Comment:
Why are you setting the `element.style.height` 2 times? First with this 1px?

Reply:
Don't know, copy paste from the internet 🙈